src/makeRobotAccountant.js

```diff
@@ -1,12 +1,24 @@
 'use strict';
 
-/**
- *
- * @return {function}
- */
-
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  let errEveryEven = false;
+
+  return function(num1) {
+    return function(num2) {
+      if (count >= 3) {
+        errEveryEven = !errEveryEven;
+      } else {
+        count++;
+      }
+
+      if (errEveryEven) {
+        return 'Bzzz... Error!';
+      }
+
+      return num1 + num2;
+    };
+  };
```

Comment:
```suggestion
  let count = 0;

  return function(num1) {
   count++
    return function(num2) {
      if (count >= 3 && count%2 === 0) {
        return 'Bzzz... Error!';
      } 
      return num1 + num2;
    };
  };
```
you can simplify this